tests/scripts/check-files.py

```diff
@@ -155,6 +155,12 @@ def __init__(self, log_file):
             ".c", ".h", ".sh", ".pl", ".py", ".md", ".function", ".data",
             "Makefile", "CMakeLists.txt", "ChangeLog"
         )
+        self.excluded_directories = ['.git', 'mbed-os']
+        self.excluded_paths = list(map(os.path.normpath, [
+            'cov-int',
+            'examples',
+            'yotta/module'
```

Comment:
The commit message said `yotta/modules`. Should this be `module` or `modules`? Update here or the commit message to be correct.

Reply:
It's `module`. I amended the last commit to fix the commit message.

Reply:
The PR to remove yotta support will be merged before this one. I don't think you need that line.

Reply:
And as an aside - there's no need to block it because `check-files.py` passes that directory in our repo anyway.